pyaedt/edb_core/edb_data/components_data.py

```diff
@@ -534,6 +534,21 @@ def center(self):
         center = cmpinst.GetCenter()
         return [center.X.ToDouble(), center.Y.ToDouble()]
 
+    @property
+    def bounding_box(self):
+        """Return the component bounding box.
```

Comment:
```suggestion
        """Component's bounding box.
```
Description for a property should be a noun string.